sympy/matrices/expressions/transpose.py

```diff
@@ -50,8 +50,8 @@ def arg(self):
     def shape(self):
         return self.arg.shape[::-1]
 
-    def _entry(self, i, j, expand=False):
```

Comment:
Although you know that the expand should not be passed on, how do you know that this argument shouldn't be here in the `def _entry` line?

Reply:
All the tests of travis are passed without this argument also.

Reply:
Now added in declaration but removed in call.